### ISSUE TITLE
Build java bindings with native M1 instruction set

### ISSUE THREAD

**Issue by aymanhab** [opensim-org/opensim-core#3938]
Since GUI can't compile a full stack native M1 libraries as of 10/24, we will publish/support API builds with native M1 libraries.

**Comment by aymanhab** [comment]
https://drive.google.com/file/d/1N4fqu0PWnEL8aKSTTs8DRsSPTLW6rAkN/view?usp=drive_link

**Comment by aymanhab** [comment]
The above link has API build with java bindings but without Moco.  I'll post one with Moco soon.

**Comment by aymanhab** [comment]
Build with moco libraries (on osx 15) https://drive.google.com/file/d/12y27vA40tL7WxqpaU5jEY0DvE2wNEG-n/view?usp=sharing

**Comment by aymanhab** [comment]
@mattpetrucci Here's the zip files for api builds on M1

**Comment by nickbianco** [comment]
@aymanhab is this in response to #3917?

**Comment by aymanhab** [comment]
Not really, the issue in 3917 happened even using Rosetta and in python. This responds to bj wanting to run api natively in matlab on Apple silicon

**Comment by nickbianco** [comment]
Gotcha. Is this something we should PR into core?

**Comment by aymanhab** [comment]
Sure, the only thing to PR would likely be a script to wire matlab to an
API release the rest is just a build process.

**Comment by mattpetrucci** [comment]
Tried running `configureOpenSim.m`. Selected the main folder and it breaks showing the message "The Folder you selected is not an OpenSim installation." Did the same when I select the bin folder.

It looks like the problem is in line 95 of the code. It wants the path to get the `openSimFolder`, which needs the `appDir` which is this path: /Users/Matt/Documents/OpenSimArtifacts/opensim-core-source-install/opensim-core-source-install.app. This path doesn't exist.

**Comment by aymanhab** [comment]
Thanks Matt, indeed this is expected as the script never used or tested an api install, I'll update and let you know next week. Thanks again
